File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int max_ones = 0;
        int current_ones = 0;
        for(int i = 0; i < nums.size(); i++) {
            if(nums[i] == 1) {
                current_ones++;
                if(current_ones > max_ones) {
                    max_ones = current_ones;
                }
            } else {
                current_ones = 0;
            }
            cout << current_ones << endl;
        }
        return max_ones;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        #Make hashmap
        freq_map = {}
        for n in nums:
            if n in freq_map:
                freq_map[n] += 1
            else:
                freq_map[n] = 1
        
        ord_map = {k: v for k, v in sorted(freq_map.items(), key=lambda item : item[1], reverse=True)}

        return list(ord_map.keys())[0:k]